src/services/NotificationService.ts

```diff
@@ -36,12 +37,14 @@ class NotificationService {
     }
 
     if (emailType === EMAIL_TYPE.ATF) {
-      console.log(`report successfully sent to ATF for PNumber ${params.testStationPNumber} with activity ${activityId}.`);
+      console.log(`report successfully created promise for ATF for PNumber ${params.testStationPNumber} with activity ${activityId}.`);
     } else if (emailType === EMAIL_TYPE.VSA) {
-      console.log(`report successfully sent to VSA for PNumber ${params.testStationPNumber} with activity ${activityId}.`);
+      console.log(`report successfully created promise for VSA for PNumber ${params.testStationPNumber} with activity ${activityId}.`);
     }
 
     return Promise.all(sendEmailPromise).catch((error: ServiceException) => {
+      // @ts-ignore
+      console.log(`failed to send for ${JSON.stringify(sendEmailPromise)}`);
```

Comment:
do we know what the response of sendEmailPromise actually looks like?

Reply:
Nope